spec.html

```diff
@@ -12620,12 +12623,12 @@ <h1>
             1. If _extensible_ is *false*, return *false*.
             1. If _O_ is *undefined*, return *true*.
             1. If ! IsAccessorDescriptor(_Desc_) is *true*, then
-              1. Create an own accessor property named _P_ of object _O_ whose [[Get]], [[Set]], [[Enumerable]], and [[Configurable]] attribute values are described by _Desc_. If the value of an attribute field of _Desc_ is absent, the attribute of the newly created property is set to its <emu-xref href="#table-object-property-attributes">default value</emu-xref>.
+              1. Create an own accessor property named _P_ of object _O_ whose [[Get]], [[Set]], [[Enumerable]], and [[Configurable]] attributes are set to the value of the corresponding field in _Desc_ if it has the field, otherwise to the attribute's <emu-xref href="#table-object-property-attributes">default value</emu-xref>.
```

Comment:
```suggestion
              1. Create an own accessor property named _P_ of object _O_ whose [[Get]], [[Set]], [[Enumerable]], and [[Configurable]] attributes are set to the value of the corresponding field in _Desc_ if it has the field, and to the attribute's <emu-xref href="#table-object-property-attributes">default value</emu-xref> otherwise.
```

Reply:
Okay. I wonder if "or" would be better than "and" though.

Reply:
"and" seems unambiguously incorrect; "or" seems like the right choice here

Reply:
I'm fine with either.

Reply:
I've changed "and" to "or" per @ljharb, and squashed the fixup commit.